### PR TITLE
memory manager: move to beta

### PR DESCRIPTION
**Pull request by cynepco3hahue**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

Move the memory manager feature to beta.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
Move the memory manager feature to the beta that will make it enabled by default with the None policy.
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
[KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1769-memory-manager
```

Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

File: pkg/features/kube_features.go
Comment (mrunalp) [inline pkg/features/kube_features.go:128]: Shouldn't this be `v1.22`?

Comment (cynepco3hahue) [reply pkg/features/kube_features.go:128]: Good catch! It should be,
```
// alpha: v1.21
// beta: v1.22
```

**Comment by cynepco3hahue** [comment]
/remove-priority important-longterm
/priority important-soon

Advance priority because the graduation was approved.

**Comment by pacoxu** [comment]
Updating the KEP links in the description would be appreciated.
- https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1769-memory-manager

**Comment by cynepco3hahue** [comment]
> Updating the KEP links in the description would be appreciated.
> 
> * https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1769-memory-manager

Hi Paco, what links do you want to update?

**Comment by pacoxu** [comment]
image

Something like this.
- [KEP]: <link>


```
#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

<!--
This section can be blank if this pull request does not require a release note.

When adding links which point to resources within git repositories, like
KEPs or supporting documentation, please reference a specific commit and avoid
linking directly to the master branch. This ensures that links reference a
specific point in time, rather than a document that may change over time.

See here for guidance on getting permanent links to files: https://help.github.com/en/articles/getting-permanent-links-to-files

Please use the following format for linking documentation:
- [KEP]: <link>
- [Usage]: <link>
- [Other doc]: <link>
-->
```

**Comment by cynepco3hahue** [comment]
@pacoxu Thanks for the clarificaiton! Done.

**Review by mrunalp — APPROVED** [review]

**Comment by ehashman** [comment]
I can approve but looks like we need https://github.com/kubernetes/kubernetes/pull/101030 to land first.

**Comment by cynepco3hahue** [comment]
@mrunalp @ehashman @matthyx Hi folks, can I have the final review round, the relevant metrics PR is merged and I can un-hold this one.

**Review by ehashman — APPROVED** [review]
Reviewed KEP: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1769-memory-manager#phase-2-beta-target-v122

Beta graduation criteria:

- [x] Extend E2E test coverage. [this and next PR]
- [x] Provide memory manager metrics under pod resources API. [#101030]
- [x] Feature gate is enabled by default. [this PR]
- [ ]  Provide beta-level documentation. [not required in k/k]

/lgtm
/approve
for pkg/features

**Pull request merged**
